site-manage/site-manage-tool/tool/src/webapp/vm/sitesetup/chef_site-siteInfo-list.vm

```diff
@@ -383,7 +383,7 @@
 					<thead>
 						<tr>
 							<th>$tlang.getString("group.title")</th>
-							#if ($!viewRoster)
+							#if ($!viewRoster || $viewMembershipGroups.size())
```

Comment:
This is a little strange, I think it would make more sense to the reader if you used isEmpty:

```suggestion
							#if ($!viewRoster || !$viewMembershipGroups.isEmpty())
```